### DISCUSSION TITLE
Module 'Logboard' has no member named 'with'

### DISCUSSION THREAD

**Discussion by je3f0o** [HaishinKit/HaishinKit.swift#1078] [Q&A] [answered]
Hello,

Few months ago I tried examples and it was working fine. I thought I can use this project later.
So recently I'm little bit more serious about and creating an app has a feature with live streaming.
Now I wanted to add HaishinKit to my existing project and cannot figure it out how to solve this issue.
I'm using XCode package manager to add package HaishinKit from github. It's automatically added Logboard as a dependency.
Then I go to my `App.swift` file and try to create a global logger instance like in the examples.

```swift
let logger = Logboard.with("com.haishinkit.HaishinKit.iOSSwiftUI")
```

Then I got this error message
```
Module 'Logboard' has no member named 'with'
```

I tried to figure it out how to solve this issue by myself and spent a whole night.
Then I writing this question before I go to sleep.
Can you help me how to install HaishinKit in my existing project?
Or maybe I don't even need a Logboard instance anyway. I haven't tried it without logger yet...
It's just weird to me. I was comparing build phases, library frameworks etc... from example project and my existing project.
And still cannot figure it out...

Thank you.

**Comment by shogo4405** [comment, marked as answer]
I renamed `Logboard.with` to `LBBoard.with`.  See also https://github.com/shogo4405/Logboard/pull/15.
Please try.
`let logger = LBBoard.with("com.haishinkit.HaishinKit.iOSSwiftUI")`
